rmw_fastrtps_shared_cpp/src/TypeSupport_impl.cpp

```diff
@@ -377,12 +378,12 @@ const TypeObject * GetMinimalObject(
   // DDS document)
   eprosima::fastcdr::Cdr ser(
     fastbuffer, eprosima::fastcdr::Cdr::LITTLE_ENDIANNESS,
-    eprosima::fastcdr::Cdr::DDS_CDR);  // Object that serializes the data.
+    eprosima::fastcdr::CdrVersion::XCDRv1);  // Object that serializes the data.
```

Comment:
Should we set the enconding flag here?

Reply:
This is following what is done by Fast DDS Gen [here](https://github.com/eProsima/Fast-DDS-Gen/blob/master/src/main/java/com/eprosima/fastdds/idl/templates/TypeObjectSource.stg#L169-L172), so I don't know.

Reply:
This one also